test/app/features/help_center/presentation/audios/audios_page_test.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:penhas/app/features/help_center/presentation/audios/audios_page.dart';
+
+void main() {
+  // testWidgets('Audios Page', (tester) async {
```

Comment:
@pedrox-hs blz? To tentando mocar o state pra testar o widget AudiosPage. Sabe como deve ser feito? Todos os outros testes de widget estão comentados no repo.

Reply:
Tb tentei usar como referência o exemplo de testes do [flutter_modular](https://modular.flutterando.com.br/docs/flutter_modular/test/).

Reply:
@shonorio Teria alguma sugestão?

Reply:
@Edely, estava dando uma olhada aqui e você pode fazer algo +/- assim:

```dart
import 'package:flutter/material.dart';
import 'package:flutter_modular/flutter_modular.dart';
import 'package:flutter_modular_test/flutter_modular_test.dart';
import 'package:flutter_test/flutter_test.dart';
import 'package:mocktail/mocktail.dart';
import 'package:penhas/app/features/help_center/domain/states/audios_state.dart';
import 'package:penhas/app/features/help_center/presentation/audios/audios_controller.dart';
import 'package:penhas/app/features/help_center/presentation/audios/audios_page.dart';
import 'package:flutter_mobx/flutter_mobx.dart';

import '../../../../../utils/aditional_bind_module.dart';

void main() {
  late AudiosController mockController;

  setUp(() {
    enableWarnWhenNoObservables = false;
    mockController = _MockAudiosController();
    
    when(() => mockController.loadPage()).thenAnswer((_) => Future.value());

    initModule(
      AditionalBindModule(
        binds: [
          Bind.instance<AudiosController>(mockController),
        ],
      ),
    );
  });

  testWidgets('Audios Page', (tester) async {
    // arrange
    when(() => mockController.currentState)
        .thenReturn(const AudiosState.initial());

    // act
    await tester.pumpWidget(
      const MaterialApp(
        home: AudiosPage(),
      ),
    );

    // assert
    // assertions for initial state
  });
}

class _MockAudiosController extends Mock implements AudiosController {}
```